androidsdk/src/main/java/com/wootric/androidsdk/views/phone/SurveyLayoutPhone.java

```diff
@@ -326,7 +327,9 @@ private void setTexts() {
         mAnchorNotLikely.setText(mSettings.getAnchorNotLikely());
         mBtnSubmit.setText(mSettings.getBtnSubmit());
         mBtnDismiss.setText(mSettings.getBtnDismiss());
-        mBtnEditScore.setText(mSettings.getBtnEditScore());
+        //mBtnEditScore.setText(mSettings.getBtnEditScore());
+        mBtnEditScore.setText("BACK");
```

Comment:
Remove this. Leave `mBtnEditScore.setText(mSettings.getBtnEditScore());` as we will change this on eligibility server.